src/test/java/uk/ac/ebi/eva/pipeline/io/VepProcessTest.java

```diff
@@ -80,26 +85,32 @@ private String getVariantInVepInputFormat(EnsemblVariant ensemblVariant) {
 
     @Test
     public void testWorkflowFlushWithoutOpening() throws Exception {
-        VepProcess vepAnnotationFileWriter = new VepProcess(annotationParameters, CHUNK_SIZE, VEP_TIMEOUT);
+        StringWriter writer = new StringWriter();
+        boolean skipComments = true;
+        VepProcess vepProcess = new VepProcess(annotationParameters, CHUNK_SIZE, VEP_TIMEOUT, writer, skipComments);
 
         exception.expect(IllegalStateException.class);
-        vepAnnotationFileWriter.flush();
+        vepProcess.flush();
     }
 
     @Test
     public void testWorkflowFlushAfterClosing() throws Exception {
-        VepProcess vepAnnotationFileWriter = new VepProcess(annotationParameters, CHUNK_SIZE, VEP_TIMEOUT);
+        StringWriter writer = new StringWriter();
+        boolean skipComments = true;
+        VepProcess vepProcess = new VepProcess(annotationParameters, CHUNK_SIZE, VEP_TIMEOUT, writer, skipComments);
 
-        vepAnnotationFileWriter.open();
-        vepAnnotationFileWriter.close();
+        vepProcess.open();
+        vepProcess.close();
 
         exception.expect(IllegalStateException.class);
-        vepAnnotationFileWriter.flush();
+        vepProcess.flush();
     }
 
     @Test
     public void testWorkflowCloseWithoutOpening() throws Exception {
-        VepProcess vepAnnotationFileWriter = new VepProcess(annotationParameters, CHUNK_SIZE, VEP_TIMEOUT);
-        vepAnnotationFileWriter.close();
+        StringWriter writer = new StringWriter();
+        boolean skipComments = true;
+        VepProcess vepProcess = new VepProcess(annotationParameters, CHUNK_SIZE, VEP_TIMEOUT, writer, skipComments);
+        vepProcess.close();
     }
```

Comment:
Well, I see the class is tested not directly, but from VepAnnotationProcessorTest, right?